plugins/processors/geo/README.md

```diff
@@ -0,0 +1,32 @@
+# Geo Processor Plugin
```

Comment:
Rename to `s2geo` or similar.

Reply:
Will the Flux `experimental/geo`  package also be renamed to `experimental/s2geo`?